src/test/java/it/pagopa/pn/bff/pnclient/delivery/PnDeliveryClientPAImplTest.java

```diff
@@ -65,4 +73,70 @@ void getSentNotificationV23Error() {
                 UserMock.PN_CX_GROUPS
         )).expectError(PnBffException.class).verify();
     }
+
+    @Test
+    void searchSentNotification() {
+        when(senderReadWebApi.searchSentNotification(
+                Mockito.anyString(),
+                Mockito.any(it.pagopa.pn.bff.generated.openapi.msclient.delivery_web_pa.model.CxTypeAuthFleet.class),
+                Mockito.anyString(),
+                Mockito.any(OffsetDateTime.class),
+                Mockito.any(OffsetDateTime.class),
+                Mockito.anyList(),
+                Mockito.anyString(),
+                Mockito.any(NotificationStatus.class),
+                Mockito.anyString(),
+                Mockito.anyString(),
+                Mockito.anyInt(),
+                Mockito.anyString()
+        )).thenReturn(Mono.just(notificationSentMock.getNotificationSentPNMock()));
+
+        StepVerifier.create(pnDeliveryClientPAImpl.searchSentNotification(
+                UserMock.PN_UID,
+                it.pagopa.pn.bff.generated.openapi.msclient.delivery_web_pa.model.CxTypeAuthFleet.PA,
+                UserMock.PN_CX_ID,
+                OffsetDateTime.now(),
+                OffsetDateTime.now(),
+                UserMock.PN_CX_GROUPS,
+                "RECIPIENT_ID",
+                NotificationStatus.ACCEPTED,
+                "SUBJECT",
+                "IUN",
+                10,
+                "NEXT_PAGES_KEY"
+        )).expectNext(notificationSentMock.getNotificationSentPNMock()).verifyComplete();
+    }
+
+    @Test
+    void searchSentNotificationError() {
+        when(senderReadWebApi.searchSentNotification(
+                Mockito.anyString(),
+                Mockito.any(it.pagopa.pn.bff.generated.openapi.msclient.delivery_web_pa.model.CxTypeAuthFleet.class),
+                Mockito.anyString(),
+                Mockito.any(OffsetDateTime.class),
+                Mockito.any(OffsetDateTime.class),
+                Mockito.anyList(),
+                Mockito.anyString(),
+                Mockito.any(NotificationStatus.class),
+                Mockito.anyString(),
+                Mockito.anyString(),
+                Mockito.anyInt(),
+                Mockito.anyString()
+        )).thenReturn(Mono.error(new WebClientResponseException(404, "Not Found", null, null, null)));
+
+        StepVerifier.create(pnDeliveryClientPAImpl.searchSentNotification(
+                UserMock.PN_UID,
+                it.pagopa.pn.bff.generated.openapi.msclient.delivery_web_pa.model.CxTypeAuthFleet.PA,
+                UserMock.PN_CX_ID,
+                OffsetDateTime.now(),
+                OffsetDateTime.now(),
+                UserMock.PN_CX_GROUPS,
+                "RECIPIENT_ID",
```

Comment:
Define this and other parameters into the notification mock as done for UserMock